src/Raven.Studio/typescript/models/database/settings/connectionStringSqlEtlModel.ts

```diff
@@ -9,7 +9,7 @@ import sqlConnectionStringSyntax = require("common/helpers/database/sqlConnectio
 class connectionStringSqlEtlModel extends connectionStringModel {
 
     static sqlProviders: Array<valueAndLabelItem<string, string> & { tooltip: string }> = [
-        { value: "System.Data.SqlClient", label: "Microsoft SQL Server", tooltip: sqlConnectionStringSyntax.mssqlSyntax },
+        { value: "Microsoft.Data.SqlClient", label: "Microsoft SQL Server", tooltip: sqlConnectionStringSyntax.mssqlSyntax },
```

Comment:
I think that we need to leave the old entry for backward compatibility and add a new one. CC @ml054 

Also see https://issues.hibernatingrhinos.com/issue/RavenDB-22365/Confusing-message-for-SQL-ETL-after-upgrade-from-5.4.115-to-118